nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -400,7 +432,8 @@ nav_msgs::msg::Path SmacPlannerHybrid::createPlan(
   if (orientation_bin >= static_cast<float>(_angle_quantizations)) {
     orientation_bin -= static_cast<float>(_angle_quantizations);
   }
-  _a_star->setGoal(mx_goal, my_goal, static_cast<unsigned int>(orientation_bin));
+  _a_star->setGoal(mx_goal, my_goal, static_cast<unsigned int>(orientation_bin),
+                  getGoalHeadingMode(), getCoarseSearchResolution());
```

Comment:
I think you can remove these getters (ditto for lattice)

Reply:
done